spot-electron/config.js

```diff
@@ -0,0 +1,24 @@
+const process = require('process');
+
+module.exports = {
+
+    /**
+     * The default URL to connect to.
+     */
+    defaultSpotURL: process.env.SPOT_URL || 'https://spot.8x8.vc/spot',
+
+    /**
+     * The details of the spash screen, if any.
+     */
+    spashScreen: {
+        height: 192,
+        logo: 'static/applogo.png',
```

Comment:
Is this the jitsi logo?

Reply:
Can be, not now. Yana said we should use this for now

Reply:
Me no likey. It doesn't make sense that this is not the Jitsi logo. There is no precedent, even.

Reply:
Removed, as well as replaced the default URL